src/layout/Error/Error.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Error.scss';
+
+export default () => (
+  <>
+    <div className="container af-container--notfound">
```

Comment:
there is already 404 page notfound

Reply:
It's just a declination of the 404 page for a 500 page